syntax = "proto3";

option go_package = "github.com/MartinSimango/goftqproto/internal/response/genresponse";

package response;

import "ftqproto/ResponseFile.proto";
import "ftqproto/ResponseStatus.proto";

message CreateResponse {
    repeated ResponseFile files = 1;
    ResponseStatus status = 2;
    
}